modules/attachment/main.tf

```diff
@@ -16,9 +16,30 @@ data "ibm_scc_profiles" "scc_profiles" {
 }
 
 locals {
+  # Get profiles by name matching that provided in var.profile_name
+  relevant_profiles = [
```

Comment:
trying to follow the logic... this should return 1 profile right? so maybe rename it to `relevant_profile`

Reply:
No, it returns all the profile versions for the listed profile such as "IBM Cloud Financial Services". We then take that list and recreate the list from lowest to latest versions and the latest version will be the last in the list so we just index the last element and take that profile.

Reply:
it returns all profiles for that 1 profile thought right? So all versions with the profile name "IBM Cloud Financial Services" ?